DMD2_Timer.cpp

```diff
@@ -209,6 +214,22 @@ static bool unregister_running_dmd(BaseDMD *dmd)
   return still_running;
 }
 
+// ESP8266 RAM cached ISR handler, because loading procedures from flash for an ISR is a bad idea
+#if defined (ESP8266)
+static void inline ICACHE_RAM_ATTR scan_running_dmds()
```

Comment:
Rather than duplicating the body of scan_running_dmds() here, can you make a separate ISR function that checks the flash mapping and then calls scan_running_dmds()?

This would reduce the amount of copy-paste duplicate code.